src/scss/initial/_links.scss

```diff
@@ -1,46 +1,75 @@
 @use '../../../node_modules/accoutrement/sass/tools';
 
-// General Focus Styles
-// --------------------
+/// # Link Defaults
+/// Initial global defaults for links…
+/// @group links
 
+// Focus
+// -----
+/// By default, all elements get a dotted outline on focus.
+/// In practice, we can override this when other focus styles exist.
+/// @group links
 :focus {
   outline: medium dotted currentColor;
   outline-color: var(--outline-color, var(--text, currentColor));
   outline-offset: var(--quarter-shim);
 }
 
-// Link Defaults
-// -------------
-
+// Links
+// -----
+/// Basic link (and link-states) apply action and interaction text colors.
+/// Text underlines are also applied,
+/// but the target depends on the presence or absence of icons,
+/// in order to avoid awkward-looking icon underlines.
+/// @example njk
+///   {% import 'utility.macros.njk' as utility %}
+///   <a href="#">simple text link</a>
+///   {{ utility.icon_link(
+///     icon='icons/external.svg',
+///     text='link with icon',
+///     url='#'
+///   ) }}
+/// @group links
 a {
   &:link,
-  &:visited,
-  &.icon-text {
-    @include tools.transition('underline');
-    --underline-color--default: var(--ccs-accent);
+  &:visited {
+    --underline-color--default: var(--accent);
     --underline-thickness--default: 0.1em;
```

Comment:
I don't think Chrome supports `text-decoration-thickness` - so I'd guess their thick/thin lines are related to font size & weight rather than our explicit settings?

The variables can make it hard to inspect. I find myself using the search feature inside the inspector more…